hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestOzoneConfigurationFields.java

```diff
@@ -100,7 +100,9 @@ private void addPropertiesNotInXml() {
         ReconServerConfigKeys.RECON_OM_SNAPSHOT_TASK_INTERVAL_DELAY,
         ReconServerConfigKeys.RECON_OM_SNAPSHOT_TASK_FLUSH_PARAM,
         OMConfigKeys.OZONE_OM_RATIS_SNAPSHOT_AUTO_TRIGGER_THRESHOLD_KEY,
-        OMConfigKeys.OZONE_OM_HA_PREFIX
+        OMConfigKeys.OZONE_OM_HA_PREFIX,
+        OMConfigKeys.OZONE_OM_ADMIN_PROTOCOL_MAX_RETRIES_KEY,
```

Comment:
As these are just used for admin operations, I don't think tuning these would make much of a difference. It would still be configurable but just not exposed via ozone-default.xml.

Reply:
I think it would be good to add in ozone-default.xml.
So that users knows about these configs, otherwise now they need to know only by reading code.

Reply:
Because atleast internally I have seen a bug for admin commands, where using defaults it is taking longer, the users needs some settings to control retry behavior. (This is for SCM, and in SCM admin commands are part of normal client/block protocol, as in OM we have separated it out that is good.)